src/CoupledSimulations/coupled_simulation.jl

```diff
@@ -1,11 +1,38 @@
-export CoupledSimulation, step!, run!
+export CoupledSimulation, step!, run!, name
 
 """
-    CoupledSimulation
+    AbstractSimulation
 
-An abstract type representing a coupled simulation.
+An abstract type representing a model simulation.
 """
-abstract type CoupledSimulation end
+abstract type AbstractSimulation end
+
+abstract type AbstractAtmosSimulation  <: AbstractSimulation end
```

Comment:
It may not be necessary to have abstract subtypes of `AbstractSimulation`, but we'll see.